src/data_storage.py

```diff
@@ -4,75 +4,65 @@
 import sys
 from plotter import Plotter
 
+class RunStoreHandle:
```

Comment:
It's Handler not Handle, isn't it?

Reply:
It's a handle. Similarly to the terminology used by files, not as robust though

Reply:
Ok, didn't read code thorougly enough